modules/nw-externalip-about.adoc

```diff
@@ -122,6 +122,11 @@ When configuring policy restrictions, the following rules apply:
 * If `allowedCIDRs[]` and `rejectedCIDRs[]` are both set, then `rejectedCIDRs[]` has precedence over `allowedCIDRs[]`.
 * If `allowedCIDRs[]` is set, creating a `Service` object with `spec.ExternalIPs[]` will succeed only if the specified IP addresses are allowed.
 * If `rejectedCIDRs[]` is set, creating a `Service` object with `spec.ExternalIPs[]` will succeed only if the specified IP addresses are not rejected.
++
+[NOTE]
+====
+You must specify an empty `allowedCIDRs[]` in the `policy` object for `spec.ExternalIPs[]` to succeed.
```

Comment:
this is not correct, it should be "policy.rejectedCIDRs[] must be used with policy.allowedCIDRs[] together in externalIP policy in order for spec.ExternalIPs[] to succeed.

Reply:
I believe this also implies the same? This note is for the below step where it says that we should be setting an empty allowedCIDRs.
`If `rejectedCIDRs[]` is set, creating a `Service` object with `spec.ExternalIPs[]` will succeed only if the specified IP addresses are not rejected.`